core/src/main/java/com/linecorp/armeria/common/auth/OAuth1aToken.java

```diff
@@ -109,6 +115,9 @@ public static OAuth1aToken of(Map<String, String> params) {
 
     private final Map<String, String> params;
 
+    @Nullable
+    private String headerValue;
+
     private OAuth1aToken(Map<String, String> params) {
```

Comment:
I don't think the factory of this class is user-friendly. 
```java
public static OAuth1aToken of(Map<String, String> params) {
   return new OAuth1aToken(params);
}
```

How about adding a builder for `OAuth1aToken`?

Reply:
Sounds good to me. How about in a separate PR?

Reply:
Oops, I missed this comment. Yeah, let me do it the separate PR.